src/libraries/Microsoft.PowerFx.Core/Public/Types/OptionSetValueType.cs

```diff
@@ -72,9 +72,11 @@ internal override void DefaultExpressionValue(StringBuilder sb)
 
             if (info != null && info.DisplayNameProvider.LogicalToDisplayPairs.Any())
             {
+                var firstOrderedValue = info.DisplayNameProvider.LogicalToDisplayPairs.OrderBy(x => x.Key.Value).First();
```

Comment:
probably should need to specify StringComparer.Ordinal as the 2nd arg to OrderBy, just to be clear